docs/extras/use_cases/extraction/index.mdx

```diff
@@ -12,13 +14,106 @@ Examples of this include:
 - Extracting multiple rows to insert into a database from a long document
 - Extracting the correct API parameters from a user query
 
-This work is extremely related to [output parsing](/docs/modules/model_io/output_parsers/).
-Output parsers are responsible for instructing the LLM to respond in a specific format.
-In this case, the output parsers specify the format of the data you would like to extract from the document.
-Then, in addition to the output format instructions, the prompt should also contain the data you would like to extract information from.
+## Overview
+
+In LangChain we provide a few useful abstractions that help you leverage OpenAI [function calling](https://openai.com/blog/function-calling-and-other-api-updates) (`-0613` models) and which go a long way to avoid model hallucination. This is in general the best go-to to extract structured data from text using langchain. However there's other two options that you should also consider:
```

Comment:
i believe that the latest gpt-4 and gpt-3.5 models come with functions, don't need to be -0613 specifically anymore

Reply:
also, now Anthropic has function so it seem we can broader the story beyond OAI.